errors/names.go

```diff
@@ -21,23 +21,23 @@ import (
 const (
 	// RetryError occurs when checks fail after exhausting all retry attempts.
```

Comment:
ignore this file, it will disappear once previous https://github.com/ossf/scorecard/pull/712 and https://github.com/ossf/scorecard/pull/718 are merged